contracts/smartwallet/CustomSmartWallet.sol

```diff
@@ -301,14 +301,12 @@ contract CustomSmartWallet is IForwarder {
                 "initialize call in logic failed"
             );
 
-            bytes memory logicCell = abi.encodePacked(logic);
```

Comment:
This change was needed in order to run the custom logic. Without it, the address retrieved from storage was corrupted, maybe due to a dangling pointer or so.